rules/config

```diff
@@ -69,7 +69,7 @@ ENABLE_ORGANIZATION_EXTENSIONS = y
 #SONIC_PROFILING_ON = y
 
 # ENABLE_SYSTEM_TELEMETRY - build docker-sonic-telemetry for system telemetry support
-# ENABLE_SYSTEM_TELEMETRY = y
+ ENABLE_SYSTEM_TELEMETRY = y
```

Comment:
one extra space in front of the line

Reply:
Removed in https://github.com/Azure/sonic-buildimage/pull/2354/commits/c9aeb557bc9a145fc168613de555f9f929f49ff9. Thanks for the catch!